src/components/soknadsStatus/SoknadsStatusUtenInnsyn.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import Panel from "nav-frontend-paneler";
+import {Innholdstittel, Normaltekst} from "nav-frontend-typografi";
+import "./soknadsStatus.less";
+import {UrlResponse} from "../../redux/innsynsdata/innsynsdataReducer";
+import EksternLenke from "../eksternLenke/EksternLenke";
+import Lastestriper from "../lastestriper/Lasterstriper";
+
+import {REST_STATUS, skalViseLastestripe} from "../../utils/restUtils";
+import DokumentSendt from "../ikoner/DokumentSendt";
+import DatoOgKlokkeslett from "../tidspunkt/DatoOgKlokkeslett";
+
+interface Props {
+    restStatus: REST_STATUS;
+    tidspunktSendt: string | null;
+    navKontor: string | null;
+    filUrl: UrlResponse | null;
+}
+
+const SoknadsStatusUtenInnsyn: React.FC<Props> = ({restStatus, tidspunktSendt, navKontor, filUrl}) => {
```

Comment:
Tar gjerne i mot tips for hvordan dette kan skrives penere 😛

Reply:
Jeg er veldig fan av å skrive typedefinisjonene direkte, eks:
```
const SoknadsStatusUtenInnsyn = (props: { restStatus: REST_STATUS, ...}) => {}
```
Men absolutt innafor dette også. :)